index.js

```diff
@@ -1,159 +1,98 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "extends": "stylelint-config-standard",
-    "customSyntax": "scss",
-    "plugins": [
-        "stylelint-scss"
-    ],
-    "ignoreFiles": [
-        "sass/vendor/*.scss"
-    ],
-    "rules": {
-        "at-rule-empty-line-before": [
-            "always",
+    'extends': 'stylelint-config-standard-scss',
+    'customSyntax': 'postcss-scss',
+    'rules': {
+        'alpha-value-notation': 'number',
+        'at-rule-empty-line-before': [
+            'always',
             {
-                "except": [
-                    "blockless-after-same-name-blockless",
-                    "first-nested"
-                ],
-                "ignore": [
-                    "after-comment"
-                ],
-                "ignoreAtRules": [
-                    "extend",
-                    "include",
-                    "else",
-                    "elseif",
-                    "content"
+                except: ['blockless-after-blockless', 'first-nested'],
```

Comment:
I still would go here with `blockless-after-same-name-blockless` as something like:

```scss
.button {
    @include shadow();
    @include other();
    
    color: red;
}
```

is coming usecase and here I don't see why any space should be forced for include, extends, ...

Reply:
@alexander-schranz I'll remove this rule as the definition [here](https://github.com/stylelint-scss/stylelint-config-standard-scss/blob/main/index.js#L6) looks good to me. I've added a test for your example and it works.